docs/topics/api/v4_frozen/abuse.rst

```diff
@@ -0,0 +1,224 @@
+=============
+Abuse Reports
+=============
+
+.. note::
+
+    These APIs are not frozen and can change at any time without warning.
```

Comment:
Should this disclaimer remain? I thought the point of this patch is to document that `v4` *is* frozen. This comment applies to all of the files added for `v4_frozen`.

Reply:
Yeah, I didn't change much of the content of the v4_frozen docs (beyond changing the links).  I guess I should drop those disclaimers (although we didn't change the v3_legacy doc disclaimers - they still say their experimental)

Reply:
I think it makes sense to drop them as they are contrary to the point of having this frozen v4 API.

Reply:
+1 on dropping them and removing them from v3 if they are still there as well

Reply:
https://github.com/mozilla/addons-server/pull/16303/commits/97c1654fb9e739eca3fd3ff3d297eaa054090a30

Reply:
https://github.com/mozilla/addons-server/pull/16303/commits/535e1a8f75139541ecf8b3e4eb635a55e68e389a for the v3 banners too (I changed them to warnings for v3)